Library/Homebrew/bundle/adder.rb

```diff
@@ -9,6 +9,7 @@ module Bundle
     module Adder
       module_function
 
+      sig { params(args: String, type: Symbol, global: T::Boolean, file: T.nilable(String)).void }
```

Comment:
```suggestion
      sig { params(args: String, type: Symbol, global: T::Boolean, file: String).void }
```
Would be nice to avoid the `T.nilable` here where possible